planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -912,6 +912,10 @@ std::vector<Pose> StartPlannerModule::searchPullOutStartPoseCandidates(
     pull_out_lanes, start_pose.position, parameters_->th_moving_object_velocity,
     backward_path_length, std::numeric_limits<double>::max());
 
+  const auto front_stop_objects_in_shoulder_lanes = filterStopObjectsInPullOutLanes(
```

Comment:
```suggestion
  const auto front_stop_objects_in_pull_out_lanes = filterStopObjectsInPullOutLanes(
```

pull out lane can be non shoulder lane

Reply:
thank you!
changed in https://github.com/autowarefoundation/autoware.universe/pull/5983/commits/130d0d902847cc0c4da1ee805e1b3c65401001b9